doc/tools/make_rst.py

```diff
@@ -569,7 +591,7 @@ def make_rst_class(class_def, state, dry_run, output_dir):  # type: (ClassDef, S
             default = property_def.default_value
             if default is not None and property_def.overrides:
                 ref = ":ref:`{1}<class_{1}_property_{0}>`".format(property_def.name, property_def.overrides)
-                ml.append((type_rst, property_def.name, default + " (overrides " + ref + ")"))
+                ml.append((type_rst, property_def.name, default + " " + translate("(overrides %s)") % ref))
```

Comment:
I used the `"%s" % string` syntax instead of `"{}".format(string)` as I think translators will already be familiar with the `%s` substitution token (even though it's not used in the class reference POT file otherwise as it's all BBCode).